controller/api/internal_apis.go

```diff
@@ -224,6 +224,39 @@ type RESTK8sNvAcceptableAlerts struct {
 	OtherAlerts              map[string]string `json:"other_alerts"`              // key is md5 of the English message
 }
 
+type RESTNvAlerts struct {
+	NvUpgradeInfo            *RESTCheckUpgradeInfo   `json:"neuvector_upgrade_info"`
+	AcceptableAlerts         *RESTNvAcceptableAlerts `json:"acceptable_alerts,omitempty"` // acceptable controller-generated alerts
+	AcceptedAlerts           []string                `json:"accepted_alerts,omitempty"`   // keys of accepted manager-generated/user alerts
+}
+
+type RESTNvAcceptableAlerts struct {
+	ClusterRoleAlerts        *RESTNvAlertGroup `json:"clusterrole_alerts,omitempty"`
+	ClusterRoleBindingAlerts *RESTNvAlertGroup `json:"clusterrolebinding_alerts,omitempty"`
+	RoleAlerts               *RESTNvAlertGroup `json:"role_alerts,omitempty"`
+	RoleBindingAlerts        *RESTNvAlertGroup `json:"rolebinding_alerts,omitempty"`
+	NvCrdSchemaAlerts        *RESTNvAlertGroup `json:"neuvector_crd_alerts,omitempty"`
+	CertificateAlerts        *RESTNvAlertGroup `json:"certificate_alerts,omitempty"`
+	OtherAlerts              *RESTNvAlertGroup `json:"other_alerts,omitempty"`
+}
+
+type AlertType string
+
+const (
+	RBAC           AlertType = "RBAC"
+	TlsCertificate AlertType = "TLS_CERTIFICATE"
+)
```

Comment:
Optional: Add a prefix or suffix so these constants can be distinguished from other types?

Reply:
Changed it to "RESTNvAlertType" to make it consistent.